packages/api-explorer/src/ApiExplorer.tsx

```diff
@@ -116,6 +127,26 @@ export const ApiExplorer: FC<ApiExplorerProps> = ({
     return () => unregisterEnvAdaptor()
   }, [])
 
+  useEffect(() => {
+    // reconcile local storage state with URL or vice versa
+    if (initialized) {
+      const sdkLanguage = searchParams.get('sdk')
+      if (sdkLanguage && getLanguageByAlias(sdkLanguage)) {
+        // sync store with URL
+        setSdkLanguageAction({
+          sdkLanguage: sdkLanguage.toLowerCase(),
+        })
+      } else {
+        // sync URL with store
+        const sdk =
+          getAliasByLanguage(selectedSdkLanguage) || selectedSdkLanguage
```

Comment:
note that this was needed because we changed what's stored in `sdkLanguage` to be the alias. As a result, existing customers with the full language name in `sdkLanguage` would run into issues if we didn't account for this. If we change `sdkLanguage` to `sdkLanguageAlias` this will no longer be an issue.

Reply:
Noting the new rename here, we should be able to remove this check now?